Dockerfile

```diff
@@ -1,11 +1,12 @@
 ## Multistage build
-FROM golang:1.22.5-alpine as build
+FROM golang:1.23-alpine as build
 ENV CGO_ENABLED=0
 ENV GOOS=linux
 ENV GOARCH=amd64
 
 WORKDIR /src
 COPY . .
+WORKDIR /src/pkg/ext-proc
```

Comment:
Temp. I'll move this dockerfile back into the ext-proc dir when I'm finished. Just needed the root context to see all local packages (the API)